spec/unit/resources/osl_packagecloud_repo.rb

```diff
@@ -0,0 +1,51 @@
+require 'spec_helper'
+
+describe 'osl_packagecloud_repo' do
+  recipe do
+    osl_packagecloud_repo 'varnishcache/varnish60lts'
+
+    osl_packagecloud_repo 'varnishcache/varnish40' do
+      action :remove
+    end
+  end
+
+  ALL_PLATFORMS.each do |p|
```

Comment:
Instead of iterating over these in the same `context` block, you should check on `platform 'centos'` and `platform 'debian'` in separate `context`s to follow the new spec format.

Reply:
Thanks for the spec format tips!